File: typescript/divisors.ts
function getDivisors(n: number): number {
    if (n == 1) return 1
    let divisors = 1
    for (let i = 1; i <= n/2; i++)
        if (n % i == 0)
            divisors++
    return divisors
}

// driver code
console.log("divisors: " + getDivisors(4324320))

File: dart/divisors.dart
import "dart:math" show sqrt;

int getDivisors(n) {
  int divisors = 0;
  for (int i = 1; i <= sqrt(n); i++)
    if (n % i == 0)
      divisors++;
  return divisors * 2;
}

// driver code
void main() {
  print("divisors: " + getDivisors(4324320).toString());
}
